modules/localgov_directories_venue/localgov_directories_venue_or/localgov_directories_venue_or.info.yml

```diff
@@ -0,0 +1,11 @@
+name: 'LocalGov Directories Venue Open Referral'
+type: module
+description: 'Makes venue directory entries Open Referral services.'
+package: LocalGov Drupal
+core: 8.x
+core_version_requirement: ^8 || ^9
+dependencies:
+  - localgov_directories:localgov_directories_venue
+  - localgov_directories:localgov_directories_org
+  - localgov_directories:localgov_directories_or
+  - localgov_openreferral:localgov_openreferral
```

Comment:
Perhaps move this module out of the `localgov_directories_venue` directory so it's in the `modules` dir so it's easier to find and more consistent.